source/mainnet/smart-contracts/references/schema-json.rst

```diff
@@ -245,6 +249,50 @@ type JSON string. Example:
 
    { "contract": "my_contract", "func": "my_receive" }
 
+``ULeb128``
+-----------
+
+Supplied as a JSON string containing an unsigned integer. Example:
+
+.. code-block:: json
+
+   "1234567890"
+
+``ILeb128``
+-----------
+
+Supplied as a JSON string containing a signed integer. Example:
+
+.. code-block:: json
+
+   "1234567890"
+
+or
+
+.. code-block:: json
+
+   "-1234567890"
+
+``ByteList``
+------------
+
+Supplied as a JSON string containing a variable sized list of bytes encoded in lowercase hex. Example:
+
+.. code-block:: json
+
+   "1234567890abcdef"
+
+
+``ByteArray``
+-------------
+
+Supplied as a JSON string containing a fixed sized list of bytes encoded in lowercase hex. Example of ``ByteArray(8)``:
```

Comment:
```suggestion
Supplied as a JSON string containing a fixed-sized list of bytes encoded in lowercase hex. Example of ``ByteArray(8)``:
```

Reply:
Consider being more explicit about what the `8` means. I.e., that it is the number of bytes and each byte is represented by two hexadecimal numbers.

Reply:
I have added more to this